docs/topics/mta-web-assessing-apps.adoc

```diff
@@ -6,34 +6,60 @@
 [id="mta-web-assessing-apps_{context}"]
 = Assessing applications
 
-You can use the {ProductName} ({ProductShortName}) {WebName} to determine the risks involved in containerizing an application.
+You can use the {ProductFullName} to determine the risks involved in containerizing an application.
+
+A built-in questionnaire is provided for assessing the risks involved in containerizing an application. For more details, see xref:mta-default-questionnaire_user-interface-guide[Default questionnaire].
+
+Custom questionnaires can also be used. For more details, see xref:mta-custom-questionnaire_user-interface-guide[Custom questionnaire].
 
 [NOTE]
 ====
-The procedure described below uses a built-in questionnaire for assessing the risks involved in containerizing an application. By default, this procedure is required prior to reviewing the application. However, you might want to skip this step and use your own questionnaire instead. In that case, use the Skipping assessment option.
+The procedure described below uses a built-in questionnaire for assessing the risks involved in containerizing an application.
+By default, this procedure is required before reviewing the application.
+However, you might want to skip this step and use your questionnaire instead.
+In that case, use the Skipping assessment option.
 ====
 
 .Procedure
 
 . In *Migration* view, click *Application inventory*.
 +
-// ![](/Tackle2/AppAssessAnalyze/AnalysisSelect.png)
+
 . Select the application you want to assess.
 +
-[NOTE]
-====
-Only one application can be assessed at a time.
-====
-. Click  *Assess*.
+
+. In the {ProductShortName} user interface, select the *Migration* working mode.
+. Click *Application Inventory* in the left menu bar. A list of all the available applications appears in the main pane.
+. Select an application from the list that appears in the main pane, click the Options menu {kebab} at the right end of the row, and select *Assess* from the drop-down menu.
 +
-// ![](/Tackle2/AppAssessAnalyze/AssessDetails.png)
+.{ProductShortName} application assessment
+
+image::mta-web-assessment-app-01.png[Application Inventory]
 
+. From the list of available questionnaires, click *Take* for one of the questionnaires.
 . Select *Stakeholders* and *Stakeholder groups* from the lists to track who contributed to the assessment for future reference.
+
 +
 [NOTE]
 ====
-You can add *Stakeholder Groups* or *Stakeholders* on the *Controls* screen of *Migration* view.
+You can add *Stakeholder Groups* or *Stakeholders* on the *Controls* pane of *Migration* view. See section xref:mta-web-seeding-instances_user-interface-guide[Seeding an instance] for information how to add stakeholders and stakeholder groups.
 ====
```

Comment:
Future improvement -- point to the exact sections that discuss adding Stakeholders and Stakeholder Groups.